sherpa/astro/xspec/__init__.py

```diff
@@ -196,7 +201,7 @@ def __init__(self, filename, name='xstbl', parnames=(),
 
         # make translation table to turn reserved characters into '_'
         bad = string.punctuation+string.whitespace
```

Comment:
Should this line not be changed too? 

It's not clear to me why this commit the problem (e.g. `xstablemodel.tmod` != `tablemodel.tmod` at https://travis-ci.org/sherpa/sherpa/jobs/147759623#L5056) since I would have expected to see AttributeError's instead. Or are the errors being caught and ignored somewhere?